static/src/javascripts/projects/commercial/modules/header-bidding/prebid/price-config.spec.ts

```diff
@@ -1,7 +1,11 @@
-import { priceGranularity } from './price-config';
+import { criteoPriceGranularity, priceGranularity } from './price-config';
 
 describe('priceGranularity', () => {
-	test('should have correct number of buckets', () => {
+	test('default should have correct number of buckets', () => {
 		expect(priceGranularity.buckets.length).toBe(2);
 	});
+
+	test('criteo should have correct number of buckets', () => {
+		expect(criteoPriceGranularity.buckets.length).toBe(3);
```

Comment:
Is it worth adding a deep equals on the actual expected buckets? This would fail any changes to the granularities?

Reply:
Good point. Do you think it's worthwhile having a test that does a deep equal on an object that's hard-coded?

Reply:
I've gone ahead and made this change, feels a lot sturdier